tests/api.spec.ts

```diff
@@ -274,8 +290,8 @@ describe('Requests to public API', () => {
       if (response.type === 'error') return;
       expect(response.value.status !== AnalysisStatus.failed).toBeTruthy();
 
-      if (response.value.status === AnalysisStatus.complete && response.value.type === 'sarif' ) {
-        expect(response.value.sarif.runs[0].results).toHaveLength(12);
+      if (response.value.status === AnalysisStatus.complete && response.value.type === 'sarif') {
+        expect(response.value.sarif.runs[0].results).toHaveLength(17);
```

Comment:
question: I wonder why do we have a different length here? Shouldn't these changes not impact the result?

Reply:
we have to constantly adjust tests to changes in the engine. 
Newer version of the engine finds more issues in our test repo.

Reply:
let me create a separate PR for these adjustments

Reply:
done: https://github.com/snyk/code-client/pull/115